node-notes-cli/remove.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+const remove = () => {
+  fs.readFile('data.json', (err, data) => {
+    if (err) throw err;
+    const fileData = JSON.parse(data);
+    delete fileData.notes[process.argv[3]];
```

Comment:
I'd probably be a good idea to let the user know if they try to delete something that isn't there.